odoo/fields.py

```diff
@@ -524,6 +524,9 @@ def _setup_related_full(self, model):
         # determine the chain of fields, and make sure they are all set up
         target = model
         for name in self.related:
+            if name not in target._fields:
```

Comment:
Instead of an extra lookup in `target._fields`, how about using `target._fields.get(name)` below and raising if the result is `None`? It's not critical but it's not more code, so why make 2 lookups when we can make only 1?